lib/MusicBrainz/Server/ControllerBase/WS/2.pm

```diff
@@ -414,6 +415,7 @@ sub linked_recordings
 
         my @acns = map { $_->artist_credit->all_names } @$recordings;
         $c->model('Artist')->load(@acns);
+        $c->model('ArtistType')->load(map { $_->artist } @acns);
```

Comment:
Same here (though I think the `uniq_by { $_->id }` below should be standardized to just `uniq`, as above).